assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -94,9 +96,22 @@ function DashboardSummaryWidget( { Widget, WidgetReportZero, WidgetReportError }
 				...dateRangeArgs,
 				gm: 'monetizableImpressions',
 			} ),
+			isAdblockerActive: select( STORE_NAME ).isAdBlockerActive(),
 		};
 	} );
 
+	if ( isAdblockerActive ) {
+		return (
+			<Widget className="googlesitekit-dashboard-adsense-stats mdc-layout-grid">
+				<div className="mdc-layout-grid__inner">
+					<div className="mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
```

Comment:
Please, use `<Grid>`, `<Row>`, and `<Cell>` components instead of adding classes.

Reply:
In the end it looked like these weren't needed so I removed them...